docs/posts/2025-08/article2.md

```
@@ -0,0 +1,76 @@
+
+公历 1098 年，罗德岛本舰，精英干员活动室，阿斯卡纶和同样重伤初愈的 Logos 作短暂告别。最初的源石已抵达卡兹戴尔，罗德岛不日启程，Logos 将继续留守本舰，而她将在卡兹戴尔逗留，协助新议长安顿巴别塔旧部。例行寒暄和公事交接终了，两方都没什么着重的情绪，冗长的战争结束，在更加漫长的、令人心力交瘁的重建与善后途中，余下更多的是被鲜血与汗水浸透的虚无。
+
+活动室内只有他们，交谈停止后，气氛稍稍显得空荡。阿斯卡纶向来寡言，但他们大概好一段时间都不会再见面了，她忽然觉得，两个萨卡兹在阔别良久的故土多说几句也无妨。于是她提起了往事。回忆着久远得扭曲的特雷西斯的语调，她背诵了那段提卡兹箴言。“你听说过这段文字吗？”她问。
+
+出乎她的意料，Logos 思索片刻，竟然颔首肯定，“小时候听母亲讲过。”提及在河谷度过的时光，女妖之主的眉眼都温柔地舒展开，露出一点点克制又暗含自豪的笑意，“毕竟她也是六英雄之一。”
+
+阿斯卡纶当即追问：“这位‘如影随形的来宾’，它是谁？”
+
+和过去沉默相对的魔王一样，Logos 没有马上回答，他的笑容也消融在透过落地舷窗涌入的午后日光里。阳光是非静止的河川，骨笔在他指尖匀速优雅地自转着，和光线并行，权能在其间隐秘而毋庸置疑地流淌，他望着她低声说：“是死亡。”
+
+![](./res/illustration/文章配图（落下一片河水）.webp) {.centering}
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)